crates/typst-kit/Cargo.toml

```diff
@@ -24,6 +24,7 @@ native-tls = { workspace = true, optional = true }
 once_cell = { workspace = true }
 tar = { workspace = true, optional = true }
 ureq = { workspace = true, optional = true }
+fd-lock = { workspace = true }
```

Comment:
please move this upwards so that it is sorted alphabetically

Reply:
Oops, sorry. Since it's still WIP and I wanted just to make it work at _some_ level, I didn't bother with placement of this.